main/clamav/APKBUILD

```diff
@@ -4,7 +4,7 @@
 # Maintainer: Carlo Landmeter <clandmeter@gmail.com>
 pkgname=clamav
 pkgver=0.99.4
-pkgrel=0
+pkgrel=1
```

Comment:
Why the empty bump?  This already had split scripts enabled.  Rebuilding should be a no-op.

Reply:
That's indeed an error, probably fucked up during a rebase here.